test/workers/pr/changelog/index.spec.ts

```diff
@@ -1,17 +1,15 @@
 import { api } from '../../../../lib/platform/github/gh-got-wrapper';
+import * as hostRules from '../../../../lib/util/host-rules';
+import { getChangeLogJSON } from '../../../../lib/workers/pr/changelog';
+// import * as _releaseNotes from '../../../../lib/workers/pr/changelog/release-notes';
 
 jest.mock('../../../../lib/platform/github/gh-got-wrapper');
 jest.mock('../../../../lib/datasource/npm');
 
-/** @type any */
-const ghGot = api.get;
+const ghGot: jest.Mock<Promise<{ body: unknown }>> = api.get as never;
+// const releaseNotes = _releaseNotes;
 
-const hostRules = require('../../../../lib/util/host-rules');
-
-const { getChangeLogJSON } = require('../../../../lib/workers/pr/changelog');
-const releaseNotes = require('../../../../lib/workers/pr/changelog/release-notes');
-
-releaseNotes.addReleaseNotes = jest.fn(input => input);
+// releaseNotes.addReleaseNotes = jest.fn(input => input);
```

Comment:
@rarkins @JamieMagee What do you think about this change? Mock has not worked before, if i enable now, we need new tests for `addReleaseNotes`, because it is no longer tested.

Reply:
I'm lost. If the mock works then we need more tests?

Reply:
Yes, if the mock works, `addReleaseNotes` is never tested. So no coverage then.

Reply:
So i would prefer to not mock

Reply:
ok